docs/content/preview/launch-and-manage/monitor-and-alert/metrics-overview.md

```diff
@@ -0,0 +1,319 @@
+| `rocksdb_numfiles_in_singlecompaction` | files | counter | The number of files in any single compaction. |
+
+### Memtable
+
+Memtable is the first level of data storage where data is stored when you start inserting. It provides statistics about reading documents, which are essentially columns in the table. If a memtable is full, the existing memtable is made immutable and stored on disk as an SST file.
+
+Memtable has statistics about reading documents, which essentially are columns in the table.
+
+A description of key metrics in this category is listed in the following table:
+
+| Metrics | Unit | Type | Description |
+| :------ | :--- | :--- | :---------- |
+| `rocksdb_memtable_compaction_micros` | microseconds | counter | The total time in microseconds to compact an set of SST files. |
+| `rocksdb_memtable_hit` | keys | counter | The number of memtable hits. |
+| `rocksdb_memtable_miss` | keys | counter | The number of memtable misses. |
+
+These metrics are available per tablet and can be aggregated across the entire cluster using appropriate aggregations.
+
+### Write-Ahead-Logging (WAL)
+
+The Write Ahead Log (or WAL) is used to write and persist updates to disk on each tablet. The following table includes metrics which allow observing the performance of the WAL component:
+
+| Metrics | Unit | Type | Description |
+| :------ | :--- | :--- | :---------- |
+| `log_sync_latency` | microseconds | counter | The number of microseconds spent to fsync the WAL entries to disk. |
+| `log_append_latency` | microseconds | counter | The number of microseconds spent on appending a batch of values to the WAL. |
+| `log_group_commit_latency` | microseconds | counter | The number of microseconds spent on committing an entire group. |
+| `log_bytes_logged`| bytes | counter | The number of bytes written to the WAL after the tablet starts. |
+| `log_reader_bytes_read` | bytes | counter | The number of bytes read from WAL after the tablet start. |
+
+These metrics are available per tablet and can be aggregated across the entire cluster using appropriate aggregations.
+
+## YB-Master metrics
+
+The [YB-Master](../../../architecture/concepts/yb-master/) hosts system metadata, records about tables in the system and locations of their tablets, users, roles, permissions, and so on. YB-Masters are also responsible for coordinating background operations such as schema changes, handling addition and removal of nodes from the cluster, automatic re-replication of data on permanent failures, and so on.
+
+A description of key metrics in this category is listed in the following table:
+
+| Metrics | Unit | Type | Description |
+| :------ | :--- | :--- | :---------- |
+| `handler_latency_yb_master_MasterClient_GetTabletLocations` | microseconds | counter | The number of microseconds spent on fetching the replicas from the master servers. This metric includes the number of times the locations of the replicas from the master server. |
```

Comment:
the number of times the locations of the replicas are what from the master server?